test/distributed_graph_generator_test_utils.h

```diff
@@ -493,14 +500,10 @@ class dist_cdag_test_context {
 
 	void maybe_print_graphs() {
 		if(test_utils::print_graphs) {
-			test_utils::maybe_print_graph(*m_tm);
-
+			print_task_graph();
```

Comment:
This is currently a no-op.

```suggestion
			CELERITY_INFO("Task graph:\n\n{}\n", print_task_graph());
```

IMO it is suboptimal that "printing" refers both to "pure" stringification and loggin.